torq.sh

```diff
@@ -255,7 +262,7 @@ elif [[ "$2" == "-debug" ]]; then
 elif [[ "$#" -eq 0 ]]; then                                                                             
   usage
 else
-  echo "Invalid argument(s)"
+  echo "ERROR - Invalid argument(s)"
```

Comment:
Line 230 has `ERROR: `, this line has `ERROR - `. Can you pick one and use it for all error messages?